Documentation/network/pod-mac-address.rst

```diff
@@ -0,0 +1,69 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _pod_mac_address:
+
+************************************
+Use a Specific MAC Address for a Pod
+************************************
+
+
+Some applications bind software licenses to network interface MAC addresses.
+Typically, the solution is very complex and requires things like Docker within
+the cluster.
+
+Therefore, cilium provides the ability to specific MAC addresses
+for pods instead of letting the operating system allocate them.
+
+
+
+
+Specific MAC Address
+#####################
+
+The MAC address configured by the annotation described here will be visible
+from within the container on the eth0 interface. Since it is isolated to the
+container it will not collide with any other MAC addresses assigned to other
+pods on the same node.
```

Comment:
```suggestion
Cilium will configure the MAC address for the primary interface inside a
Pod if you specify the MAC address in the ``cni.cilium.io/mac-address`` before
deploying the Pod. This MAC address is isolated to the container so it will
not collide with any other MAC addresses assigned to other Pods on the same
node. The MAC address must be specified *before* deploying the Pod.
```

Reply:
Minor nit here: The naming of `eth0` for the primary interface is distribution-specific for the container image, so we should prefer `primary interface` and only use `eth0` as an example.